dockstore-client/src/test/java/io/github/collaboratory/cwl/ToilLauncherIT.java

```diff
@@ -15,14 +15,16 @@
  */
 package io.github.collaboratory.cwl;
 
+import io.dockstore.common.ToilOnlyTest;
 import org.apache.commons.io.FileUtils;
-import org.junit.Ignore;
+import org.junit.experimental.categories.Category;
+import org.junit.jupiter.api.Tag;
 
 /**
  * @author dyuen
  */
-@Ignore
-// @Category(ToilOnlyTest.class)
+@Tag(ToilOnlyTest.NAME)
```

Comment:
Something weird is going on here with junit and how this is being picked up by surefire/failsafe, may want to work on this together with updating them in general since we're out-of-date in this repo
 https://ucsc-cgl.atlassian.net/browse/SEAB-6823